src/shared/clients/lambdaClient.ts

```diff
@@ -14,4 +14,6 @@ export interface LambdaClient {
     invoke(name: string, payload?: _Blob): Promise<Lambda.InvocationResponse>
 
     listFunctions(): AsyncIterableIterator<Lambda.FunctionConfiguration>
+
+    getFunction(name: string): Promise<Lambda.GetFunctionResponse>
```

Comment:
not sure if we don't document these because they closely match the SDK, but this one is a bit non-obvious to me, so I would add a brief docstring.

```
/** Downloads the Lambda function source code as a zip. */

Reply:
FYI this doesn't do that, this just wraps the getFunction call, which returns metadata + a link to the function's zip file.

Reply:
docstring will clear that up then :)